File: KH.Pepper.Web.UI/src/app/services/api/models/userrefreshtokenDto.ts

export class UserRefreshTokenDto { 
    Id : number = 0;
    Token : string = '';
    RefreshToken : string = '';
    CreatedDate:Date = new Date();
    ExpirationDate:Date = new Date();
    IpAddress:string = '';
    IsInvalidated:boolean = false;
    UserId:  number = 0;
    
}

File: KH.Pepper.Services/DTO/UserRefreshTokenDto.cs
using AutoMapper;
using KH.Pepper.Core.Domain;

namespace KH.Pepper.Core.AppServices.Dto
{

    [AutoMap(typeof(UserRefreshToken), ReverseMap = true)]
    public class UserRefreshTokenDto
    {
         public int Id { get; set; }
        public string Token { get; set; }
        public string RefreshToken { get; set; }
        public DateTime CreatedDate { get; set; }

        public DateTime ExpirationDate { get; set; }
         
        public string IpAddress { get; set; }
        public bool IsInvalidated { get; set; }

        public int UserId { get; set; }
         
    }
}
